tsconfig.json

```diff
@@ -24,10 +24,11 @@
     "lib": ["ES2015"],
     "paths": {
       "@ukef/*": ["src/*"],
```

Comment:
Maybe we should delete this path, otherwise I'm concerned we'll keep adding `@ukef/modules` imports instead of `@ukef/module`

Reply:
I took it out, but it's creating more issues in ESLint related to finding the paths. I'll have a look at it more in a separate PR